src/components/GameView/GameCard.vue

```diff
@@ -226,6 +225,8 @@ export default {
 }
 .target-overlay {
   cursor: pointer;
+  background-color: rgb(var(--v-theme-accent-lighten1));
+  opacity: .6;
```

Comment:
Overlays no longer seem to have direct color controlling props, so I'm applying it w/ css